units/host_allocator.go

```diff
@@ -202,10 +202,20 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	// and the hosts allocated for them,
 	// how long will it take the current fleet of hosts, plus the ones we spawned, to chew through
 	// the scheduled tasks in the queue?
+
 	scheduledDuration := distroQueueInfo.ExpectedDuration - distroQueueInfo.DurationOverThreshold
 
+	var totalOverdueInTaskGroups, CountDurationOverThresholdInTaskGroups int
```

Comment:
just a style thing: the first letter ('c') in CountDurationOverThresholdInTaskGroups should probably be lower case for consistency